Lecture4/src/test/java/CalcUnitTests.java

```diff
@@ -0,0 +1,115 @@
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalcUnitTests {
+
+	@BeforeClass
+	public void oneTimeSetUp() {
+		System.out.print("Before Class");
+	}
+
+        @Test(dataProvider = "testAddDataProvider")
+        public void checkSum(double operand1, double operand2, double total) {
+            Calculator calculator = new Calculator(operand1, Operator.PLUS, operand2);
+            System.out.println("Test Total: " + calculator.makeCalculation());
```

Comment:
you did action twice 
calculator.makeCalculation() - to local var `results` - and use

Reply:
done